ssl/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply from: "$rootDir/gradle/javaModule.gradle"
+
+archivesBaseName = 'ssl'
+group = 'io.crate'
+description = 'SSL Encryption for CrateDB'
+
+dependencies {
+    compile project(':core')
```

Comment:
Yes please - core pulls in all of ES with Lucene, etc.. would be great if we could keep this slim.